com.unity.render-pipelines.universal/Runtime/Passes/CopyDepthPass.cs

```diff
@@ -115,7 +115,7 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                 cmd.SetGlobalVector(ShaderPropertyId.scaleBiasRt, scaleBiasRt);
 
                 cmd.DrawMesh(RenderingUtils.fullscreenMesh, Matrix4x4.identity, m_CopyDepthMaterial);
-
+                cmd.SetGlobalVector(ShaderPropertyId.availableTexturesParams, new Vector4(0f, 0f, 0f, 1f));
```

Comment:
This is probably not a good enough solution if/when we want to add other texture types to this parameter, such as the opaque texture. Could we add the data to RenderingData? Other ideas welcome.